## Ejercicio1 - Utilizar operadores aritméticos
Operadores aritméticos en Python
Python proporciona operadores aritméticos comunes para que puedas realizar operaciones matemáticas en tu código. Estos incluyen las cuatro operaciones principales de suma, resta, multiplicación y división.

Exploremos cómo podemos crear un programa que pueda calcular la distancia entre dos planetas. Comenzaremos usando dos distancias de planetas: Tierra (149.597.870 km) y Júpiter (778.547.200 km).

In [32]:
PosicionTierra = 149597870
PosicionJupiter = 778547200

Realizar la operación
Con los valores obtenidos, es el momento de añadir el código para realizar la operación. Restarás el primer planeta del segundo para determinar la distancia en kilómetros. A continuación, puedes convertir la distancia del kilómetro en millas multiplicándola por 0.621.

In [33]:
Distanciaentreplanetas = (PosicionJupiter - PosicionTierra)
Distanciaenmillas = (0.621 * Distanciaentreplanetas)
print("Distancia entre planetas en km: " + str(Distanciaentreplanetas) + " km")
print("------------------------------------------------------------------------") 
print("Distancia entre planetas en millas: " + str(Distanciaenmillas) + " millas")

Distancia entre planetas en km: 628949330 km
------------------------------------------------------------------------
Distancia entre planetas en millas: 390577533.93 millas


Prueba tu proyecto
Con el código creado, ejecuta el notebook para obtener el resultado. Deberías recibir lo siguiente:

628949330
390577534

## Ejercicio 2: convierte cadenas en números y usa valores absolutos
## Crear una aplicación para trabajar con números y entrada de usuario
Con frecuencia, deberás convertir los valores de cadena en números para realizar correctamente diferentes operaciones o determinar el valor absoluto de un número.

Para crear nuestra aplicación, queremos leer la distancia del sol para dos planetas, y luego mostrar la distancia entre los planetas. Haremos esto usando input para leer los valores, int para convertir a entero y luego abs para convertir el resultado en su valor absoluto.

Lee los valores
Usando input, agrega el código para leer la distancia del sol para cada planeta, considerando 2 planetas.

In [34]:
PrimerPlaneta = input("Escriba la distancia del primer planeta (en km)")

SegundoPlaneta = input("Escriba la distancia del segundo planeta (en km)")

## Convertir a número.
Debido a que input devuelve valores de cadena, necesitamos convertirlos en números. Para nuestro ejemplo, usaremos int

In [35]:
PosPrimerPlaneta = int(PrimerPlaneta)
PosSegundoPlaneta = int(SegundoPlaneta)

## Realizar el cálculo y convertir a valor absoluto
Con los valores almacenados como números, ahora puedes agregar el código para realizar el cálculo, restando el primer planeta del segundo. Debido a que el segundo planeta podría ser un número mayor, usarás abs para convertirlo a un valor absoluto. También agregarás el código para mostrar el resultado en millas multiplicando la distancia del kilómetro por 0.621

In [36]:
DistanciaKM = abs(PosSegundoPlaneta - PosPrimerPlaneta)
DistanciaMillas = 0.621 * DistanciaKM

print("Distancia resultante en km: " + str(DistanciaKM))
print("----------------------------------------------")
print("Distancia resultante en millas: " + str(DistanciaMillas))

Distancia resultante en km: 30
----------------------------------------------
Distancia resultante en millas: 18.63
